src/authorization/WebAclAuthorizer.ts

```diff
@@ -111,18 +121,25 @@ export class WebAclAuthorizer extends Authorizer {
    * @returns A store containing the relevant acl triples.
    */
   private async getAclRecursive(id: ResourceIdentifier, recurse?: boolean): Promise<Store> {
```

Comment:
Would reduce some of the debug statements here since some can be deduced from the others. There are 4 debug statements now but I think you can cover all cases with just the initial one and the error one.

Reply:
For this specific case, I would say the tricky bit is in "deduce". From experience with node-solid-server, this particular method offers functionality that has often been the target of questions "why isn't this working". We assume that the ACL is going to look like this, have that extension, exist, etc. And then it turned out that our deductions were wrong for some reason, for instance, because the path contained a space and appending `.acl` would suddenly percent-encode that. (I wish I was [making this up](https://github.com/solid/node-solid-server/issues/1211).)

So I agree with you in general, just not for this particular routine 🙂